### PR TITLE
Change from vite to create react app to fix the code_verfier undefined issue

### PR DESCRIPTION
**Pull request by dasuni-30**

### CODE REVIEW

File: petcare-sample/b2c/web-app/petdesk/web/react/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (JayaShakthi97) [inline petcare-sample/b2c/web-app/petdesk/web/react/webpack.config.js:2]:
```suggestion
 * Copyright (c) 2024, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: petcare-sample/b2c/web-app/petdesk/web/react/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        plugins: [
+            new HtmlWebpackPlugin({
+                template: "./src/index.html"
+            })
+        ],
+        resolve: {
+            extensions: [ ".tsx", ".ts", ".js", ".json" ]
+        }
+    });
+};
```
Comment (JayaShakthi97) [inline petcare-sample/b2c/web-app/petdesk/web/react/webpack.config.js:106]: Let's add a new line at the end of file

**Review by DonOmalVindula — APPROVED** [review]

File: petcare-sample/b2c/web-app/petdesk/web/react/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
```
Comment (brionmario) [inline petcare-sample/b2c/web-app/petdesk/web/react/webpack.config.js:106]:
```suggestion
};

```

Comment (dasuni-30) [reply petcare-sample/b2c/web-app/petdesk/web/react/webpack.config.js]: Added with b72e814714e7140c9f035783c55d8f93366c042b

**Review by brionmario — APPROVED** [review]

File: petcare-sample/b2c/web-app/petdesk/web/react/config.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    "clientID": "<CONFIGURED_SPA_CLIENT_ID>",
+    "scope": [
+        "openid",
+        "profile",
+        "email"
+    ],
+    "signInRedirectURL": "http://localhost:3000",
+    "signOutRedirectURL": "http://localhost:3000",
+    "resourceServerURL": "http://localhost:9090",
+    "myAccountAppURL": ""
```
[the remark this answers is not in the document]
Comment (dasuni-30) [reply petcare-sample/b2c/web-app/petdesk/web/react/config.json:12]: This is an existing config added with https://github.com/wso2/samples-is/pull/451.

**Pull request merged**